tests/backend/core/Core/Unit/Logic/PermissionsTest.php

```diff
@@ -490,6 +489,7 @@ public function permissionsTests(): array
                     'area_procedure_adjustments_general_location',
                     'area_procedure_send_submitter_email',
                     'area_search_submitter_in_procedures',
+                    'feature_statements_fragment_consideration',
```

Comment:
Please use alphabetically order (for each of the new inserted features).
You should be able to use "Edit" -> "Sort Lines" in the IDE to sort marked lines alphabetically.